packages/libs/react-components/src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,77 @@
+import { SystemIcons } from '../Icons';
+
+import {
+  AbsoluteButton,
+  StyledButton,
+  StyledHeading,
+  StyledIconButton,
+  StyledNotification,
+  StyledText,
+} from './styles';
+
+import React, { FC } from 'react';
+
+export interface INotificationProps {
+  icon?: (typeof SystemIcons)[keyof typeof SystemIcons];
+
+  title: string;
+  description?: string;
+  displayCloseButton?: boolean;
+  expand?: boolean;
+
+  actionButtonLabel?: string;
+  actionButtonOnClick?: () => void;
+  displayActionButton?: boolean;
+
+  color?: string;
+
+  simplified?: boolean;
+}
+
+export const Notification: FC<INotificationProps> = ({
+  icon,
+  title,
+  description,
+  displayCloseButton = false,
+  actionButtonLabel,
+  actionButtonOnClick,
+  displayActionButton = false,
+  color = 'default',
+  simplified = false,
+  ...props
+}) => {
+  const Icon = icon!;
+  const isSimple =
+    simplified || (!displayActionButton && description === undefined)
+      ? true
+      : false;
```

Comment:
nit: You don't need to do the ternary since it just evaluates to true or false

` const isSimple = simplified || (!displayActionButton && description === undefined)` should suffice